christfellowshipchurch/apollos-api
# Patch for groups members wrong address showing

### About
This PR is a bug patch. When a group leader used to see a group member profile, the address showing for all members was the address of the group leader rather than the individual members, this issue has been now fixed and the group member addresses are correct. 

### Test Instructions
You can test this PR locally through the server, as seen on the screenshot. Or once the PR is approved you can go to a group editing page and open all of the group member profiles, their address should now be the correct addresses. 

### Screenshots 
![Screen Shot 2022-01-05 at 11 01 06 AM](https://user-images.githubusercontent.com/46769629/148261703-c4493894-def9-4512-800d-7707c28c667b.png)

### Closes Tickets
[CFDP-1778](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-1778)

### Related Tickets
N/A
